src/InteractionBuilder.ts

```diff
@@ -94,27 +94,38 @@ function defaultOptions(interactionOptions: IInteractionOptions): IDefinedIntera
     interactionOptions.dialogflowSpreadsheets
   ) as string[];
 
-  let platforms: ISupportedPlatforms[] =
-    arrayify(interactionOptions.platforms) || DEFAULT_INTERACTION_OPTIONS.platforms;
+  let platforms: ISupportedPlatforms[] = arrayify(
+    interactionOptions.platforms
+  ) as ISupportedPlatforms[];
 
-  if (!_.isEmpty(alexaSpreadsheets)) {
-    platforms.push("alexa");
-  }
-
-  if (!_.isEmpty(dialogflowSpreadsheets)) {
-    platforms.push("dialogflow");
-  }
+  const spreadsheetMapping = {
+    alexa: alexaSpreadsheets,
+    dialogflow: dialogflowSpreadsheets
+  };
 
-  platforms = _.uniq(platforms);
+  platforms = _.chain(spreadsheetMapping)
```

Comment:
Can you leave a comment on what this snippet of code does? without data I can't tell what's the output

Reply:
Done!